### PR TITLE
Some suggestions from Grammarly

### PR DESCRIPTION
**Pull request by leo-ship-it**
Grammarly complained with some sentences in the whitepaper ;)

https://app.grammarly.com/

### CODE REVIEW

File: src/frontend/src/common.tsx [hunk 1]
```diff
@@ -974,6 +974,7 @@ export const icrcTransfer = async (
     } catch (e) {
         return "Transfer failed";
     }
+    return response;
```
Comment (taggrx) [inline src/frontend/src/common.tsx]: This is wrong, this variable is undefined here.

**Review by xqxpx — COMMENTED** [review]
Thanks so much @leo-ship-it! This was long overdue!

Please fix the JS bug and we can merge.

File: src/frontend/src/common.tsx (hunk 1 above)
Comment (leo-ship-it) [reply src/frontend/src/common.tsx]: My bad.

**Comment by mechaquan** [comment]
Btw, on the pop up message when trying to transfer tokens it says you voted on an open proposal, right?  I think this should be changed as well.

I'm not sure where this message exists so I can't confirm the exact wording but I think it should be clarified.  Simple "Tokens cannot be transferred while there is an open proposal."

**Comment by leo-ship-it** [comment]
@mechaquan Currently the error message you get is: `Transfer failed. One reason might be that you voted on a proposal that is still open.`.

I think the solution would be to better handle the error messages: https://github.com/TaggrNetwork/Taggr/pull/6/files

**Comment by mechaquan** [comment]
So you mean like more specific error responses?  Do we know what other possibilities are?  Low balance, open proposal, etc... what else?

`Transfer failed with error: ` + response ... is there a list of what these responses are?  Sorry I need GitHub for dummies...

**Comment by leo-ship-it** [comment]
So the error you get is of type:
```
type TransferError = variant {
  GenericError : GenericError;
  BadFee : BadFee;
  CreatedInFuture : CreatedInFuture;
  TooOld;
  InsufficientFunds : InsufficientFunds;
};
```

The response you get is of type one of those. 
Generic error can be one of:
- minting invariant violation
- only minting transactions are allowed without a fee
- transfers locked: a vote on a pending proposal detected
- memo longer than 32 bytes

**Review by xqxpx — APPROVED** [review]

**Comment by xqxpx** [comment]
Thanks Leo!

**Pull request merged**